.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
+branches:
+  only:
+    - master
```

Comment:
i think the team might benefit from also seeing their travis build status for feature branches, what do you think?